experimental/packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -456,11 +495,61 @@ export const runTests = (
       },
     ];
 
-    const runTest = (
+    const validateSpans = (
+      serverSpan: ReadableSpan,
+      clientSpan: ReadableSpan,
+      methodName: string,
+      attributesValidation?: {
+          serverAttributes?: Attributes,
+          clientAttributes?: Attributes
+        }) => {
+      const validations = {
+        name: `grpc.pkg_test.GrpcTester/${methodName}`,
+        status: grpc.status.OK,
+      };
+
+      assertSpan(
+        moduleName,
+        serverSpan,
+        SpanKind.SERVER,
+        validations
+      );
+      assertSpan(
+        moduleName,
+        clientSpan,
+        SpanKind.CLIENT,
+        validations
+      );
+
+      assertPropagation(serverSpan, clientSpan);
+
+      if (attributesValidation?.clientAttributes) {
+        for (const key in attributesValidation.clientAttributes) {
+          const expected = attributesValidation.clientAttributes[key] as string;
+          const actual = clientSpan.attributes[key] as string;
+
+          assert.equal(actual, expected);
+        }
+      }
+
+      if (attributesValidation?.serverAttributes) {
+        for (const key in attributesValidation.serverAttributes) {
+          const expected = attributesValidation.serverAttributes[key];
+          const actual = serverSpan.attributes[key];
+
+          assert.equal(actual, expected);
+        }
+      }
+    };
+
```

Comment:
Consider make this function as common util function since we're going to add lot more maps to attributes :)

Reply:
Is there a common place to move it to so we can access this in other packages? or should I put it in a utils class in the grpc instrumentation only?

Reply:
I think he meant in the grpc instrumentation as its specific to it, however i believe it can be done in another PR that add more attributes map to check.

Reply:
Sounds good! We can move it when needed